lib/ui/hooks.dart

```diff
@@ -41,7 +41,11 @@ void _updateWindowMetrics(double devicePixelRatio,
 
 typedef String LocaleClosure();
 
-String _localeClosure() => window._locale.toString();
+String _localeClosure() {
+  const String enUS = 'en_US';
+  final String locale = window?._locale?.toString() ?? enUS;
```

Comment:
`?.` wasn't used before - seems like we could do without them here.  If we do need then, then you should check for null below.